skills/refactor/COMMAND.md

```
@@ -46,18 +46,18 @@ The command triggers the refactor skill which:
 3. **Brief capture**: Records problem, goals, approach in state
 4. **Execution**: Track-specific implementation
 5. **Validation**: Tests pass, goals met, docs updated
 
 ## State Initialization
 
-When invoked, initializes refactor workflow state:
+When invoked, initializes refactor workflow state using `mcp__workflow-state__workflow_init`:
 
-```bash
-~/.claude/scripts/workflow-state.sh init --refactor <feature-id>
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)